frontend/packages/component-library/src/video/YoutubeVideo.tsx

```diff
@@ -55,13 +56,15 @@ const YouTubeVideo = ({src, onError}: YouTubeVideoProps) => {
     <ReactPlayer
       height={'100%'}
       width={'100%'}
-      url={src}
-      onError={onError}
+      src={src}
+      title={videoTitle}
+      onError={onError as unknown as ReactEventHandler<HTMLVideoElement>}
       previewTabIndex={-1} // the play icon is the tabbable portion
       playing={true}
       controls={true}
+      autoPlay={true}
       config={{
-        playerVars: {autoplay: 1, rel: 0},
+        youtube: {rel: 0},
```

Comment:
Curious why `autoplay` was removed?

Reply:
`autoplay` was removed as a configuration option apparently in favor of the normal `<video>` element `autoplay` property.

The config option is set via: https://github.com/muxinc/media-elements/blob/abbd9f7e1bcf43ec49d2b69ed1b12c3b695656f4/packages/youtube-video-element/youtube-video-element.js#L62

It is not in the config property type: https://github.com/muxinc/media-elements/blob/main/packages/youtube-video-element/youtube-video-element.d.ts